thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/views/AnomalyResultView.js

```diff
@@ -210,6 +213,10 @@ AnomalyResultView.prototype = {
 
 
   },
+
+  destroy() {
+    $("#anomaly-results-place-holder").html('');
```

Comment:
remove would be better to detach the event binders.
If you'd like to keep the `#anomaly-results-place-holder` element, `.children().remove()` would be fine.